controllers/admin/AdminCartsController.php

```diff
@@ -67,8 +67,7 @@ public function __construct()
 		LEFT JOIN (
             SELECT DISTINCT `id_guest`
             FROM `' . _DB_PREFIX_ . 'connections`
-            WHERE
-                TIME_TO_SEC(TIMEDIFF(\'' . pSQL(date('Y-m-d H:i:00', time())) . '\', `date_add`)) < 1800
```

Comment:
Is there a reason, why we are using `pSQL(date('Y-m-d H:i:00', time()))` here instead of `now()` ? 
I would have simplified further to `WHERE `date_add` > date_add(now() , interval -30 minute)`

Reply:
hello @lmeyer1 , here is the reason:

https://github.com/PrestaShop/PrestaShop/issues/30828

Reply:
@matthieu-rolland That issue is closed and NOW works absolutely correctly :D

Reply:
nobody said that now doesn't work, but it uses the mysql timezone which in some cases can be different from the timezone of the shop

Reply:
@matthieu-rolland Thanks. I didn't use `now()`, so everything is OK.